statistics/histogram.go

```diff
@@ -1069,6 +1070,37 @@ func (c *Column) IsLoaded() bool {
 	return c.Loaded || c.notNullCount() > 0
 }
 
+// IsNecessaryLoaded indicates whether the necessary statistics is loaded.
+// If `IsLoaded` returns true, we will directly return due to they are directly loaded from outer storage and
+// keep the tests still correct.
+// If `IsLoaded` returns false, we will check whether histogram and topn are both loaded as the statistics need
+// at least one of them to do optimize.
+func (c *Column) IsNecessaryLoaded() bool {
```

Comment:
There seems to be little difference between `IsNecessaryLoaded` and `IsLoaded`. Can we just keep one?